string name
string state
---
bool success